x-pack/test_serverless/api_integration/test_suites/common/reporting/datastream.ts

```diff
@@ -23,35 +27,42 @@ export default function ({ getService }: FtrProviderContext) {
 
   describe('Data Stream', () => {
     before(async () => {
+      role = await svlUserManager.createApiKeyForRole('admin');
+
       await esArchiver.load(archives.ecommerce.data);
       await kibanaServer.importExport.load(archives.ecommerce.savedObjects);
 
       // for this test, we don't need to wait for the job to finish or verify the result
-      await reportingAPI.createReportJobInternal('csv_searchsource', {
-        browserTimezone: 'UTC',
-        objectType: 'search',
-        searchSource: {
-          index: '5193f870-d861-11e9-a311-0fa548c5f953',
-          query: { language: 'kuery', query: '' },
-          version: true,
+      await reportingAPI.createReportJobInternal(
+        'csv_searchsource',
+        {
+          browserTimezone: 'UTC',
+          objectType: 'search',
+          searchSource: {
+            index: '5193f870-d861-11e9-a311-0fa548c5f953',
+            query: { language: 'kuery', query: '' },
+            version: true,
+          },
+          title: 'Ecommerce Data',
+          version: '8.15.0',
         },
-        title: 'Ecommerce Data',
-        version: '8.15.0',
-      });
+        role
+      );
     });
 
     after(async () => {
-      await reportingAPI.deleteAllReports();
+      await reportingAPI.deleteAllReports(role);
       await esArchiver.unload(archives.ecommerce.data);
       await kibanaServer.importExport.unload(archives.ecommerce.savedObjects);
+      await svlUserManager.invalidateApiKeyForRole(role);
     });
 
     it('uses the datastream configuration with set ILM policy', async () => {
```

Comment:
Sorry, the test name is wrong as-is
```suggestion
    it('uses the datastream configuration', async () => {
```